core/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -4,9 +4,9 @@
  * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance with the License. You may obtain a
  * copy of the License at
- *
+ * 
```

Comment:
Ah okay, I think what might be happening is the formatter adds the space but some people have trailing white space removal in their editor which then removes it. In any case, we can change it all to be consistent when a style is decided on. No change needed for this PR.